bin/spark-class

```diff
@@ -45,6 +45,7 @@ fi
 if [ ! -d "$SPARK_JARS_DIR" ] && [ -z "$SPARK_TESTING$SPARK_SQL_TESTING" ]; then
   echo "Failed to find Spark jars directory ($SPARK_JARS_DIR)." 1>&2
   echo "You need to build Spark before running this program." 1>&2
+  echo "Note: In Spark 2.0 the required build target has changed from \"assembly\" to \"package\"" 1>&2
```

Comment:
This looks OK though I suppose this message can be shortened to just directly state that you need to "build Spark with target 'package' before running this program". Looks like `mvn package` still works so that's all fine.